### PR TITLE
Andres/admin section

### PR DESCRIPTION
**Pull request by andres3months**
## Ch, Ch, Ch, Ch, Changes...

- Fixes dotenv issues
- updates readme
- CSS fixes and tweaks
- Layout fixes and tweaks
- Adds events activation / deactivation on admin panel
- Adds soft deletion to event

## Deploy actions

Bundle / Migrate after merging

## Checklist
Make sure these boxes are checked before submitting your contribution

- [x] This PR has only the commits related to my branch
- [x] I've done self review and my code looks great!
- [x] I've tested my changes and written tests when applicable

### CODE REVIEW

File: app/views/admin/events/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+						%th Owner Phone
+				%tbody
+					- @events.each do |event|
+						%tr
+							%td = event.name
+							%td = event.description
+							%td = event.starts_at
+							%td = event.ends_at
+							%td = event.owner_email
+							%td = event.owner_phone
```
Comment (Janther) [inline app/views/admin/events/index.html.haml]: In this company we indent with 2 spaces XD
vowaauk

Comment (andres3months) [inline app/views/admin/events/index.html.haml]: Stupid RubyMine 

Fixing

**Comment by Janther** [comment]
Dude all of your files are using tabs

**Comment by Tinusw** [comment]
Looks good to me @Janther?

File: app/views/layouts/admin.html.haml [hunk 2]
```diff
@@ -11,23 +11,17 @@
...
-    = #TODO : Experimental. Remove from here.
-
-    :scss
-      body {
-        padding-top: 5px;
-        padding-bottom: 40px;
-      }
-
-  %body
+  %body{class: "#{controller_name}_#{action_name}", 'data-controller' => controller_name, 'data_action' => action_name, id: 'admin_body' }
```
Comment (Janther) [inline app/views/layouts/admin.html.haml]: Haml can be more awesome.

```HAML
  %body#admin_body{ class: "#{controller_name}_#{action_name}",
                    data: { controller: controller_name,
                            action: action_name } }
```

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -10,21 +10,15 @@
...
-
-      %h2 
-        Application Layout
-
       = yield
-
     = javascript_include_tag 'application'
-    %script{:async => '', :defer => 'defer', :src => "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_MAPS_API_KEY']}&callback=initMap"}
\ No newline at end of file
+    %script{:async => '', :defer => 'defer', :src => "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_MAPS_API_KEY']}&callback=initMap"}
```
Comment (Janther) [inline app/views/layouts/application.html.haml]: Don't be afraid to use multiple lines on tag attributes in HAML
```HAML
%script{ async: '',
         defer: 'defer',
         src: "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_MAPS_API_KEY']}&callback=initMap" }
```

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -10,21 +10,15 @@
...
     %title WREMO
     = Gon::Base.render_data
-
-    %meta{:content => 'WREMO', :name => 'description'}/
     = stylesheet_link_tag 'application', media: 'all'
   %body
     /[if lt IE 8]
       <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-
     %div.container
```
Comment (Janther) [inline app/views/layouts/application.html.haml]: we don't need `%div` use only `.container`

File: app/views/shared/_messages.html.haml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+- flash.each do |type, message|
+  %div{ class: "#{bootstrap_class_for(type)} alert fade in"}
```
Comment (Janther) [inline app/views/shared/_messages.html.haml]: `.alert.fade.in{ class: bootstrap_class_for(type) }`

File: app/views/shared/_messages.html.haml [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+    %button.close{ 'data-dismiss'=>'alert' }
```
Comment (Janther) [inline app/views/shared/_messages.html.haml]:
```HAML
%button.close{ data: { dismiss: 'alert' } }
```

File: app/views/shared/admin/_topnav.html.haml [hunk 7]
```diff
@@ -12,7 +12,6 @@
         %li.active
           = link_to 'Events', admin_events_path
       %ul.nav.navbar-nav.navbar-right
-
         %li.dropdown
           %a{href: '#', class: 'dropdown-toggle', 'data-toggle' => 'dropdown', role: 'button', 'aria-haspopup' => 'true', 'aria-expanded' => 'false'}
```
Comment (Janther) [inline app/views/shared/admin/_topnav.html.haml]:
```HAML
%a.dropdown-toggle{ href: '#', data: { toggle: 'dropdown' }, role: 'button', aria: { haspopup: 'true', expanded: 'false' } }
```

File: app/views/admin/events/index.html.haml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.container
+  .row
+    .col-lg-12
+      %h3 Event List
+      %hr/
+      = render partial: 'shared/messages', flash: flash
+      %table#datatable.table.table-striped.table-hover
```
Comment (Janther) [inline app/views/admin/events/index.html.haml]: use a partial for the table

File: app/views/admin/events/index.html.haml [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+          %tr
+            %th Name
+            %th Description
+            %th Starts at
+            %th Ends at
+            %th Owner Email
+            %th Owner Phone
+            %th
+        %tbody
+          - @events.each do |event|
```
Comment (Janther) [inline app/views/admin/events/index.html.haml]: use a partial for the row

Comment (andres3months) [inline app/views/admin/events/index.html.haml]: Partial for tables and for rows will come in the next PR, there is some improvements I'd like to to on the DTs and in the bootstrap template before

File: app/views/admin/events/index.html.haml [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+              %td= event.name
+              %td= event.description
+              %td= event.starts_at
+              %td= event.ends_at
+              %td= event.owner_email
+              %td= event.owner_phone
+              %td.text-center
+                - if event.deleted?
+                  = link_to activate_admin_event_path(event), method: :put, data: { confirm: 'Are you sure?' }, class: 'btn btn-primary btn-sm' do
+                    Activate
```
Comment (Janther) [inline app/views/admin/events/index.html.haml]: Keep it in mind
```HAML
= link_to 'Activate', activate_admin_event_path(event),
          method: :put,
          data: { confirm: 'Are you sure?' },
          class: 'btn btn-primary btn-sm'
```

**Pull request merged**